Source/AtomicWebView/WebClient.cpp

```diff
@@ -683,6 +687,13 @@ void WebClient::SendKeyEvent(const StringHash eventType, VariantMap& eventData)
     host->SendKeyEvent(keyEvent);
 #endif
 
+#ifdef ATOMIC_PLATFORM_LINUX
+    if ( keyEvent.character == 0xD && keyEvent.type == KEYEVENT_KEYUP) 
+    {
+        keyEvent.type = KEYEVENT_CHAR;
+        host->SendKeyEvent(keyEvent);
```

Comment:
OSX has similar work arounds, the keyboard handling between SDL and CEF is tricky, as CEF deals in native keycodes and has no abstractions for OSR